### PR TITLE
Added bisect.pyi and ConfigParser.pyi

### PR DESCRIPTION
**Pull request by Dakkaron**

### CODE REVIEW

File: stdlib/2.7/ConfigParser.pyi [hunk 1]
```diff
@@ -0,0 +1,178 @@
+from typing import Any, Tuple
+
+__all__ = None # type: list[str]
+DEFAULTSECT = None # type: str
+MAX_INTERPOLATION_DEPTH = None # type: int
+
+class Error(Exception):
+    def _get_message(self): # type: () -> None
+        pass
```
Comment (matthiaskramm) [inline stdlib/2.7/ConfigParser.pyi]: Please use the notation
def _get_message(self) -> None: ...
.
Also below.

File: stdlib/2.7/ConfigParser.pyi [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+    def __repr__(self): # type () -> str
+        return ""
+
+    __str__ = __repr__
+
+
+class NoSectionError(Error):
+    def __init__(self, section): # type: (str) -> None
+        self.section = None # type: str
```
Comment (matthiaskramm) [inline stdlib/2.7/ConfigParser.pyi]: Please make these into class-level attributes:

class NoSectionError(Error):
  section = ...  # type: str
  ...

Also below.

File: stdlib/2.7/bisect.pyi [hunk 3]
```diff
@@ -0,0 +1,17 @@
+def bisect(a,x,lo=0,hi=0): # type: (list,object,int,int) -> int
```
Comment (matthiaskramm) [inline stdlib/2.7/bisect.pyi]: Same as in the other file, using PEP 3107 for this is preferred:

def bisect(a:list, x, lo:int=..., hi:int=...) -> int: ...

**Comment by gvanrossum** [comment]
Ping?

**Comment by Dakkaron** [comment]
Sorry, didn't have any time before. Fixed everything there.

File: stdlib/2.7/ConfigParser.pyi [hunk 4]
```diff
@@ -0,0 +1,140 @@
+from typing import Any, Tuple
+
+__all__ = None # type: list[str]
+DEFAULTSECT = None # type: str
+MAX_INTERPOLATION_DEPTH = None # type: int
+
+class Error(Exception):
+    def _get_message(self) -> None: ...
+
```
Comment (gvanrossum) [inline stdlib/2.7/ConfigParser.pyi]: Please disregard what your linter may tell you -- we prefer no blank lines between methods in stubs.

File: stdlib/2.7/ConfigParser.pyi [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+class RawConfigParser:
+    _dict = ... # type: Any
+    _sections = ... # type: dict
+    _defaults = ... # type: dict
+    _optcre = ... # type: Any
+    def __init__(self, defaults: dict[Any,Any] = ..., dict_type: Any = ..., allow_no_value: bool = ...) -> None: ...
+
+    def defaults(self) -> dict[Any,Any]: ...
```
Comment (gvanrossum) [inline stdlib/2.7/ConfigParser.pyi]: Same here.

File: stdlib/2.7/bisect.pyi [hunk 6]
```diff
@@ -0,0 +1,11 @@
+def bisect(a: list, x: object, lo: int, hi: int) -> int: ...
+
```
Comment (gvanrossum) [inline stdlib/2.7/bisect.pyi]: Here, too I'd skip the blank lines.

**Comment by gvanrossum** [comment]
Thanks!

**Comment by Dakkaron** [comment]
Thanks for the patience ;)

**Pull request merged**